src/popup/components/LinkButton.vue

```diff
@@ -14,6 +14,12 @@
         name="icon"
       />
     </span>
+    <span
+      v-if="isExternal"
+      class="link-icon"
+    >
+      <ExternalLinkIcon />
+    </span>
```

Comment:
Instead of having two spans with the same class and the possibility of having 2 icons at the same moment I suggest to modify existing span:

```
    <span
      v-if="isExternal || $slots.icon"
      class="link-icon"
    >
      <ExternalLinkIcon v-if="isExternal" />
      <slot
        v-else
        name="icon"
      />
    </span>
```